### ISSUE TITLE
wix310-debug.zip download is blocked by Chrome for being a "malicious" file.

### ISSUE THREAD

**Issue by trockefeller** [wixtoolset/issues#5466]
I was trying to download wix310-debug.zip from the codeplex site https://wix.codeplex.com/releases/view/624906 and Chrome blocked the download saying that "This file is malicious, and Chrome has blocked it." they don't give any details into why it was flagged as malicious, and I was able to download the file from the Edge browser successfully.

I'm not sure if this is the best place to put this, but I figured someone should know that Chrome is blocking downloads.

**Comment by robmen** [comment]
Duplicate of #5460
